klaus/contrib/app_args.py

```diff
@@ -12,6 +27,23 @@ def strtobool(val):
 
 
 def get_args_from_env():
+    """Extract Klaus application arguments and configuration from environment variables.
+
+    Returns:
+        tuple: A tuple containing (args, kwargs) where:
+            - args (tuple): Positional arguments (repos list, site name)
+            - kwargs (dict): Keyword arguments with configuration options
+
+    Environment Variables:
+        KLAUS_REPOS: Space-separated list of repository paths
+        KLAUS_SITE_NAME: Name of the site (default: "unnamed site")
```

Comment:
Please don't add default values here. They are very obvious from reading the code.